live-ingester/src/integration-test/resources/application-test.yml

```diff
@@ -11,6 +11,10 @@ alfresco:
     shared-file-store:
       retry:
         initial-delay: 0
+  filter:
```

Comment:
This could be removed as in `application.yml` it looks the same.

Reply:
Thanks. I guess it's a leftover from some of test tryouts. Removed.